common/main/en_CA.xml

```diff
@@ -1869,11 +1869,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 						<dateFormatItem id="hmv">↑↑↑</dateFormatItem>
```

Comment:
Don't we also need to change the patterns for GyMd above (and also add patterns from GyM and GyMED which we are currently inheriting from en[_US])? And what about the intervalFormats for short dates, need to fix those too.

Reply:
`GyMd` et al. wasn’t set in 41 and prior, so there was nothing to revert here. But I can revert that also.  The interval formats don't work as well because they already use dashes for ranges, thought we said we would push that to next time.

Reply:
@pedberg-icu  PTAL - see if this addresses it

Reply:
en.xml for reference:

```xml
						<dateFormatItem id="Gy">y G</dateFormatItem>
						<dateFormatItem id="GyMd">d MMM y G</dateFormatItem>
						<dateFormatItem id="GyMMM">MMM y G</dateFormatItem>
						<dateFormatItem id="GyMMMd">d MMM y G</dateFormatItem>
						<dateFormatItem id="GyMMMEd">E, d MMM y G</dateFormatItem>
```

Reply:
GyMd now changed. Did not find GyM or GyMED